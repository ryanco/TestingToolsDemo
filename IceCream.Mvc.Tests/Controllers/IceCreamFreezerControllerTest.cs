using ApprovalTests.Asp;
using ApprovalTests.Asp.Mvc;
using ApprovalTests.Reporters;
using IceCream.Mvc.Controllers;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace IceCream.Mvc.Tests.Controllers
{
	[TestClass]
	public class IceCreamFreezerControllerTest
	{
		[TestMethod]
		[UseReporter(typeof(TortoiseDiffReporter))]
		public void TestSimpleView()
		{
			PortFactory.MvcPort = 3235;
			MvcApprovals.VerifyMvcPage(new IceCreamFreezerController().SimpleView);
		}
	}
}
