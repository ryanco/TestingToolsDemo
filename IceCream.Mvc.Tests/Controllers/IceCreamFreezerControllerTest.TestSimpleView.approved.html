

<!DOCTYPE html>

<html>
	<head><base href="http://localhost:3235/IceCreamFreezer">
		<title>This is a simple view</title>
	</head>
	<body>
		<div>
			<p>Makin gud websites</p>  
		</div>
	</body>
</html>